judge/fixtures/language_all.json

```diff
@@ -942,5 +942,21 @@
         "description": "",
         "extension": "cpp"
     }
+},
+{
+    "model": "judge.language",
+    "pk": 77,
+    "fields": {
+        "key": "NODEJS",
+        "name": "JavaScript (Node.js)",
```

Comment:
This naming sounds good to me -- could we also rename V8 to "JavaScript (V8)" for consistency?